conditional/templates/errors.html

```diff
@@ -1,8 +1,10 @@
 {% extends "nav.html" %}
 {% block title %}Error {{ error_code }}{% endblock %}
 {% block body %}
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Typically, viewports are added to the `<head>` of HTML documents. You may want to move this bit up to a new [`extraHeader` block](https://github.com/ComputerScienceHouse/conditional/blob/develop/conditional/templates/base.html#L14).